src/web/app/pages-student/student-profile-page/upload-edit-profile-picture-modal/upload-edit-profile-picture-modal.component.ts

```diff
@@ -18,146 +13,115 @@ import { StudentProfilePictureResults } from '../../../../types/api-output';
   styleUrls: ['./upload-edit-profile-picture-modal.component.scss'],
 })
 export class UploadEditProfilePictureModalComponent implements OnInit {
-
-  isImageLoaded: boolean = false;
-  user: string = '';
-  fileName: string = 'No File Selected';
-  isFileSelected: boolean = false;
+  imageChangedEvent: any = '';
+  croppedImage: any = '';
+  showCropper: boolean = false;
+  imageLoadFailed: boolean = false;
```

Comment:
Correct. This was an experimental flag I was playing around with. Apologies, will remove it since it feels like it is really not needed at the moment.